readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page using grid
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_grid/)
+- [DEMO LINK](https://Alla-Karakaliuk.github.io/layout_catalog_grid/)
```

Comment:
<img width="836" alt="image" src="https://user-images.githubusercontent.com/38065883/167785617-209a1604-5b8c-4b65-a038-207ca210702f.png">

<img width="501" alt="image" src="https://user-images.githubusercontent.com/38065883/167785660-3783e4b8-5e6d-4124-89eb-406f6951df24.png">

Reply:
Thanck you. Media queries was fixed